fiatconnect-api.md

```diff
@@ -2352,6 +2353,36 @@ The `ifsc` or Indian Financial System Code is an 11-digit alpha-numeric code tha
 The `accountNumber` field is unique between individuals and no two banks or account holders can have the same account number.
 Banks use different starting codes for their branches for differentiation. (In India, bank account numbers usually contain 7 to 21 digits.)
 
+#### 9.3.2.6. `PIXAccount`
+
+`PIXAccount`  egg. [PIX](https://www.bcb.gov.br/estabilidadefinanceira/pix) is a Brazilian instant payment method created by the Central Bank (BC) in which funds are transferred between accounts in a few seconds, at any time or day. Pix transfers can be carried out from a checking account, savings account or prepaid payment account.
+
+```
+{
+  accountName: `string`,
+	institutionName: `string`,
+  keyType: `PIXKeyTypeEnum`,
+  key: `string`,
+  fiatAccountType: `FiatAccountTypeEnum.BankAccount`
+}
+```
+
+##### 9.3.2.6.1. `PIXKeyTypeEnum`
+
+`keyType` MUST be one of the following values:
+
+```
+[
+  `EMAIL`,
+  `CPF`,
+  `PHONE`,
+  `RANDOM`
+]
+```
+
+Depending on the type of key, the `key` field follows some validation rules. It MUST be an email, [CPF](https://en.wikipedia.org/wiki/CPF_number), mobile number (including area code) or a random (32 alphanumeric characters) key respectively. A random key is generated by bank.
```

Comment:
Can you provide a specific format for mobile numbers? Is [E.164](https://en.wikipedia.org/wiki/E.164) acceptable?